### ISSUE TITLE
render error on show remote data on datagrid

### ISSUE THREAD

**Issue by ZenLiuCN** [vitmalina/w2ui#927]
In My project get some grid render errors(**Env**:Php5.3,Nginx,windows,chrome)
My server response like this :
```js
{"total":13,"records":[{"id":"1","text":"\u7cfb\u7edf\u7ef4\u62a4","icon":"fa fa-cogs blue","url":"","parent_id":"0","role_level":"999","pagename":null},{..}]}
```
and it's columns set like this below(get from database):
```js
{"status":"success","message":"","recode":{"type":"Grid","data":{"name":"pmenu","columns":[{"id":"6","tab":"Menu","field":"id","caption":"ID","size":"2%","min":null,"max":null,"sortable":null,"searchable":null,"resizable":null,"hidable":null,"attr":null,"style":null,"title":null,"editable":null,"render":null},{"id":"7","tab":"Menu","field":"text","caption":"\u540d\u5b57","size":"10%","min":null,"max":null,"sortable":null,"searchable":null,"resizable":null,"hidable":null,"attr":null,"style":null,"title":null,"editable":null,"render":null},{"id":"8","tab":"Menu","field":"icon","caption":"\u56fe\u6807","size":"10%","min":null,"max":null,"sortable":null,"searchable":null,"resizable":null,"hidable":null,"attr":null,"style":null,"title":null,"editable":null,"render":null},{"id":"9","tab":"Menu","field":"url","caption":"url","size":"10%","min":null,"max":null,"sortable":null,"searchable":null,"resizable":null,"hidable":null,"attr":null,"style":null,"title":null,"editable":null,"render":null},{"id":"10","tab":"Menu","field":"parent_id","caption":"\u4e0a\u5c42","size":"10%","min":null,"max":null,"sortable":null,"searchable":null,"resizable":null,"hidable":null,"attr":null,"style":null,"title":null,"editable":null,"render":null},{"id":"11","tab":"Menu","field":"Role_level","caption":"\u6743\u9650","size":"10%","min":null,"max":null,"sortable":null,"searchable":null,"resizable":null,"hidable":null,"attr":null,"style":null,"title":null,"editable":null,"render":null},{"id":"16","tab":"Menu","field":"PageName","caption":"\u9875\u9762\u4ee3\u7801","size":"5%","min":null,"max":null,"sortable":null,"searchable":null,"resizable":null,"hidable":null,"attr":null,"style":null,"title":null,"editable":null,"render":null}],"url":{"get":"\/ajax\/getMenu","remove":"\/ajax\/delMenu","save":"\/ajax\/upMenu"},"header":"","recid":"id","summary":[],"toolbar":[]}}}
```
I got 
```js
Uncaught TypeError: Cannot read property '1' of nullw2ui-1.4.2.js:6422 
w2grid.getCellHTMLw2ui-1.4.2.js:6387 
w2grid.getRecordHTMLw2ui-1.4.2.js:6136 
w2grid.getRecordsHTMLw2ui-1.4.2.js:4681
 w2grid.refreshw2ui-1.4.2.js:3353 
w2grid.requestCompletew2ui-1.4.2.js:3231 
(anonymous function)jquery.min.js:2 
m.Callbacks.jjquery.min.js:2
 m.Callbacks.k.fireWithjquery.min.js:4
 xjquery.min.js:4 m.ajaxTransport.send.b
```
I found it's render columns with out *render* setting get error,it's from my server side out put *render* as **Null**
```js
   if (typeof col.render == 'object')   data = '<div>' + (col.render[data] || '') + '</div>';
```
Well it be better to see IS it *null* before render?

**Comment by lcz20** [comment]
Just and ** && col.render!=null** in line 6417 it's be nice
```js
 if (typeof col.render != 'undefined' && col.render!=null) {
```

**Comment by vitmalina** [comment]
Fixed in both 1.4 and master branch, you will need to download compiled version in /dist folder
